astropy/units/imperial.py

```diff
@@ -2,8 +2,15 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
 
 """
-This package defines colloquially used Imperial units.  By default, they
-are not enabled.  To enable them, do::
+This package defines colloquially used Imperial units.  They are
```

Comment:
> To use these units, you must import the `astropy.units.imperial` module::

@adrn This is actually not true (and part of the clarification).  Imperial units *are* available without importing `astropy.units.imperial`.  They just require the `u.imperial` namespace.